VideoWeb/VideoWeb/Controllers/ConferenceStatusController.cs

```diff
@@ -74,20 +73,19 @@ public async Task<IActionResult> SetVideoControlStatusesForConference(Guid confe
         [HttpGet("{conferenceId}/getVideoControlStatuses")]
         [SwaggerOperation(OperationId = "GetVideoControlStatusesForConference")]
         [ProducesResponseType(typeof(ConferenceVideoControlStatuses), (int)HttpStatusCode.OK)]
-        [ProducesResponseType(typeof(ConferenceVideoControlStatuses), (int)HttpStatusCode.NotFound)]
+        [ProducesResponseType(typeof(ConferenceVideoControlStatuses), (int)HttpStatusCode.NoContent)]
         public async Task<IActionResult> GetVideoControlStatusesForConference(Guid conferenceId)
```

Comment:
@krishnapottekkat-hmcts It needs to be NoContent because the server successfully processed the request, and is not returning any content. When you use Not Found, it is being reported as a client error which isn't the case for us. We currently see tons of client errors in the console having it as Not Found when in actual sense no error occurred processing the request.

Reply:
Thats alrite in terms of a Create, Update and Delete. For GET, you are requesting a resource and you are processing it and not returning anything does not make sense.

Reply:
@krishnapottekkat-hmcts Absolutely, we're requesting a resource and if it's not there we shouldn't treat it as a error when it isn't the case.